chibuezeayogu/React-Redux
# #161155222  Implement add course

**What does the PR do?**
- This PR Implements  `Add Course`

**Description of Task to be completed?**
- create add course form
- validate input fields
- create add course actions
- write test
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-add-course-161155222`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161155222](https://www.pivotaltracker.com/story/show/161155222)

**Screenshots**
- N/A
